cmd/kubeadm/app/cmd/upgrade/plan.go

```diff
@@ -33,6 +33,7 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/cli-runtime/pkg/printers"
 	"k8s.io/klog/v2"
+	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
```

Comment:
should be in the cmd/kubeadm group below

Reply:
Done.